src/net_processing.cpp

```diff
@@ -634,6 +634,11 @@ class PeerManagerImpl final : public PeerManager
     /** Retrieve unbroadcast transactions from the mempool and reattempt sending to peers */
     void ReattemptInitialBroadcast(CScheduler& scheduler) EXCLUSIVE_LOCKS_REQUIRED(!m_peer_mutex);
 
+    /**
+     * Private implementation of IsInvInFilter which does not call GetPeerRef; to be prefered when the PeerRef is available.
+     */
+    bool IsInvInFilter(const PeerRef& peer, const uint256& hash) const;
```

Comment:
Please avoid taking `PeerRef` as argument, internal functions throughout use `Peer` (see examples [here](https://github.com/dashpay/dash/blob/30fc76c3970371aa97aa8e7baada868162a3932d/src/net_processing.cpp#L712), [here](https://github.com/dashpay/dash/blob/30fc76c3970371aa97aa8e7baada868162a3932d/src/net_processing.cpp#L680) and [here](https://github.com/dashpay/dash/blob/30fc76c3970371aa97aa8e7baada868162a3932d/src/net_processing.cpp#L1107)).